### DISCUSSION TITLE
46

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#704] [Comments]
http://jojoldu.tistory.com/46

**Comment by jojoldu** [comment]
작성자: 김얗마

눈팅으로 꾸준히 보고 있습니다. 빠르게 마감된 세미나라 놓쳤었는데 이렇게 간접적으로 정리된 내용을 알 수 있어서 좋네요. 항상 게시한 글들 보고 도움을 얻네요. 
乃(엄지척)

**Comment by jojoldu** [comment]
작성자: 창천향로

옷 ㅠㅠㅠ 감사합니다!! 너무 힘이 나는 댓글이네요 ㅎㅎ 더 열심히 작성하도록 하겠습니다!! ㅎㅎ

**Comment by jojoldu** [comment]
작성자: 또로로

어제 계셨나보네요! 저도 적기는 했는데 저보다 보기 좋게 정리하셨네요. 공유 해주셔서 감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

좋게 봐주셔서 제가 더 감사합니다^^

**Comment by jojoldu** [comment]
작성자: 울트라

정성스러운 리뷰 감사드립니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

댓글 감사드립니다^^
좋은 세미나가 있으면 더 열심히 찾아 듣고 리뷰남기겠습니다
감사합니다!

**Comment by jojoldu** [comment]
작성자: 호박털이범

눈팅으로 후기 정말 잘 보고 있습니다. 고맙습니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

앗 제가 더 감사드립니다^^ 자주 방문 부탁드리겠습니다!

**Comment by jojoldu** [comment]
작성자: jeon

안녕하세요 보통 스프링 입문 교재 보는 것이 좋을까요?

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

안녕하세요! 개인적으로는 스프링 책을 바로보기 보다는 박재성 교수님의 Next Step 교재를 먼저 보시고 가시는 것이 낫지 않을까 생각합니다.
만약 책 내용이 조금 어렵다면,
엄진영 강사님의 열혈강의 자바 웹 개발 워크북을 먼저 보시고 보시는것을 추천드립니다.

**Comment by gonghojin** [comment]
안녕하세요~ 창천향로님 블로그에 자주 들러 좋은 글 읽고 가는 신입개발자입니다ㅎ.
댓글을 잘 남기는 성향은 아니지만, 좋은 지침이 될 글들을 많이 제공해주셔서 너무 감사하여 글을 남깁니다. 앞으로도 좋은 글 부탁드립니다!

**Comment by jojoldu** [comment]
@gonghojin 도움이 되셨다니 다행입니다 ㅎㅎ;
더 열심히 글 써야겠네요!
앞으로도 잘 부탁드리겠습니다 :)

**Comment by walbatrossw** [comment]
항상 좋은글 감사합니다 ㅎㅎ

**Comment by jojoldu** [comment]
@walbatrossw 잘봐주셔서 감사합니다 :)
